### PR TITLE
Add option to disable running default agents.

### PR DESCRIPTION
**Pull request by deadly-panda**

### CODE REVIEW

File: src/ostorlab/runtimes/local/runtime.py [hunk 1]
```diff
@@ -92,7 +92,7 @@ def __init__(
         *args,
         tracing: Optional[bool] = False,
         mq_exposed_ports: Optional[Dict[int, int]] = None,
-        gcp_logging_credential: Optional[str] = None,
+        run_default_agents: bool = True,
```
Comment (3asm) [inline src/ostorlab/runtimes/local/runtime.py:95]: Unit test?

**Review by 3asm — CHANGES_REQUESTED** [review]

File: tests/runtimes/local/runtime_test.py [hunk 2]
```diff
@@ -182,3 +186,43 @@ def docker_services():
...
+    assert start_agent_mock_call_args[0][0][0].key == "agent/ostorlab/agent42"
+    assert start_agent_mock_call_args[1][0][0].key == "agent/ostorlab/inject_asset"
+    assert all(
+        call_arg[0][0].key != "agent/ostorlab/local_persist_vulnz"
+        for call_arg in start_agent_mock_call_args
+    )
+    assert all(
+        call_arg[0][0].key != "agent/ostorlab/tracker"
+        for call_arg in start_agent_mock_call_args
+    )
```
Comment (3asm) [inline tests/runtimes/local/runtime_test.py:228]: Explicit checks.

Comment (3asm) [reply tests/runtimes/local/runtime_test.py:228]: I don't get these checks. Should not you rather check that there is no agent tracker in the list of agents?

**Review by 3asm — CHANGES_REQUESTED** [review]

File: tests/runtimes/local/runtime_test.py (hunk 2 above)
Comment (deadly-panda) [reply tests/runtimes/local/runtime_test.py:228]: That's exactly what I am doing, I'm looping over all and making sure they are all `!=` tracker and persist vulnz.

Comment (3asm) [inline tests/runtimes/local/runtime_test.py:228]:
```suggestion
    assert all(
        call_arg[0][0].key != "agent/ostorlab/local_persist_vulnz"
        for call_arg in start_agent_mock_call_args
    ) is True
    assert all(
        call_arg[0][0].key != "agent/ostorlab/tracker"
        for call_arg in start_agent_mock_call_args
    ) is True
```

**Review by 3asm — DISMISSED** [review]

**Review by 3asm — APPROVED** [review]

**Pull request merged**